jetty-core/jetty-fcgi/jetty-fcgi-proxy/src/main/config/modules/fcgi-proxy.mod

```diff
@@ -0,0 +1,13 @@
+[description]
+Enables support for HTTP to FastCGI proxying.
+
+[tags]
+fcgi
+proxy
+
+[depend]
+fcgi
+proxy
+
+[lib]
+lib/fcgi/jetty-fcgi-proxy-${jetty.version}.jar
```

Comment:
do we ship with this class in the jetty-home archive?
(wondering if we should be downloading here or not)

Reply:
Not sure what class you refer to. 
But yes the `jar` is in `jetty-home`, I have a test in `DistributionTests` that prove it.